src/cf-typography.less

```diff
@@ -1171,6 +1178,7 @@
 */
 
 .list__branded {
+    /* TODO: Why do we need .list_item when li will always be in the list */
```

Comment:
Maybe? But if we're already targeting `li`, the `.list_item` is just excess. It shouldn't ever be anything but an `li`